bin/deploy.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 rsync -r --delete-after --quiet $TRAVIS_BUILD_DIR johnny-5@138.197.20.170:devanoobot
-ssh johnny-5@138.197.20.170 "cd devanoobot && ../.npm-global/bin/pm2 restart devanoobot.config.json --update-env"
+ssh johnny-5@138.197.20.170 "cd devanoobot && npm run migrations:prod && ../.npm-global/bin/pm2 restart devanoobot.config.json --update-env"
```

Comment:
@StrangeWill Is this the only solution here? It should get the job done, but I don’t like it.

I was hoping to find a way to tell `pm2` to run a command when restarting an application from the config file, but a cursory look at [this page](http://pm2.keymetrics.io/docs/usage/application-declaration/) did not yield anything. Something like this would be awesome:
```javascript
// devanoobot.config.json
{
  "apps": [
    {
      "name": "bot",
      // … other properties
      "on_restart": "npm run migrations:prod"
    },
    // … etc.
  ]
}
```

Do you know if something like this exists?